src/credentials/providers/sharedCredentialsProvider.ts

```diff
@@ -139,6 +141,36 @@ export class SharedCredentialsProvider implements CredentialsProvider {
         return true
     }
 
+    private getProfile(name: string) {
+        return getSectionDataOrThrow(this.sections, name, 'profile')
+    }
+
+    private getSsoProfileFromProfile(profile = this.profile): SsoProfile {
```

Comment:
Is the `profile` parameter redundant here since it doesn't seem there is a need for it?

Reply:
Yup.